internal/pipe/nfpm/nfpm_test.go

```diff
@@ -425,7 +425,7 @@ func TestRunPipe(t *testing.T) {
 		case "android":
 			require.Equal(t, "foo_1.0.0_android_"+arch+"-10-20"+ext, pkg.Name)
 		case "aix":
-			require.Equal(t, "foo_1.0.0_aix_ppc-10-20"+ext, pkg.Name)
+			require.Equal(t, "foo_1.0.0_aix_ppc64-10-20"+ext, pkg.Name)
```

Comment:
This is incorrect now. As I explained in my comment, this needs to be ppc, even though the system is ppc64. The RPM arch **must** be set to ppc for AIX, regardless of the system being ppc64.

Reply:
the name template used in the test uses goarch, so this is correct. there's another test using `ConventionalFilename` below

Reply:
Okay, that makes sense. However, you're setting infoArch to ppc, which also doesn't work in my mind. I separated infoArch and RPMarch on purpose. My understanding is that infoArch needs to remain ppc64 while the rpmArch needs to be ppc.